crates/evm/src/query.rs

```diff
@@ -48,14 +48,14 @@ const ESTIMATE_GAS_ERROR_RATIO: f64 = 0.015;
 pub(crate) struct EstimatedGas {
```

Comment:
maybe we can add an explanation comment on this struct.

let's explain why we add an l1_fee_overhead to the actual estimated gas